modules/testcases/jdkAll/Eap71x/pom.xml

```diff
@@ -450,6 +450,16 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <jboss.home>${maven.multiModuleProjectDirectory}/modules/servers/eap7/build/target/jbossas</jboss.home>
```

Comment:
Is maven.multiModuleProjectDirectory set?
Could we add jboss.home system property at test-configurations (level) pom, in case in some other test configuration someone would like to set another path? (some thing similar to 117b78e)